### DISCUSSION TITLE
Selecting data points in a scatter plot

### DISCUSSION THREAD

**Discussion by flyinghigh012** [jfree/jfreechart#292] [Q&A]
I would like to select a set of data points using the resizable rectangle similar to the zoom function. Is this still not implemented for JFreeChart? If not, how would I go about implementing this?

**Comment by trashgod** [comment]
I think of this as falling within the purview of the chart container, e.g. `ChartPanel` or `ChartViewer`, rather than the chart itself. In the latter case, I see that the chart's `ChartCanvas` supports named, tiered `MouseHandlerFX` implementations, which may serve as a guide. Also consider a tabular selection, as in a check box list of model values.
